sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/csv/CSVOptions.scala

```diff
@@ -164,6 +164,10 @@ class CSVOptions(
       s"${DateFormatter.defaultPattern}'T'HH:mm:ss[.SSS][XXX]"
     })
 
+  val timestampNTZFormatInRead: Option[String] = parameters.get("timestampNTZFormat")
```

Comment:
Do `timestampNTZFormatInRead` and `timestampNTZFormatInWrite` have different behavior?

Reply:
How do you mean? Actually, it is the same config but reads require an `Option[String]` and write require `String` (similar to `timestampFormatInRead` and `timestampFormatInWrite`). I can update to keep only one property `timestampNTZFormat` instead. Would that work?

Reply:
Since we are not going to respect the conf `SQLConf.get.legacyTimeParserPolicy` here, I think we can make it one property

Reply:
Updated!

Reply:
@gengliangwang I had to revert to using `timestampNTZFormatInRead` and `timestampNTZFormatInWrite` - one of the tests verifies that timestampsNTZ are returned in a certain format. You can review 043edb627e79a3f836caff608722e90ddc3997d9.